app/controllers/dashboard/sub_service_requests_controller.rb

```diff
@@ -29,7 +29,7 @@ def index
     service_request         = ServiceRequest.find(params[:srid])
     protocol                = service_request.protocol
     @admin_orgs             = @user.authorized_admin_organizations
-    @permission_to_edit     = protocol.project_roles.find_by(identity_id: @user.id)
+    @permission_to_edit     = !protocol.project_roles.find_by(identity_id: @user.id, project_rights: ['approve', 'request']).nil?
     @sub_service_requests   = service_request.sub_service_requests
```

Comment:
Find roles where 'approve' or 'request' rights, then use !.nil? to for 'are there any?'